core/setup.py

```diff
@@ -58,7 +58,8 @@ def create_version_file():
     'boto3',
     'pandas>=1.0.0,<2.0',
     'scikit-learn>=0.22.0,<0.24',
-    'autograd>=1.3'
+    'autograd>=1.3',
+    'dill==0.3.3',
```

Comment:
it dill added to solve some unpickable object?

Reply:
This is cherry-picked from https://github.com/awslabs/autogluon/pull/767